Project.toml

```diff
@@ -10,9 +10,11 @@ Dierckx = "39dd38d3-220a-591b-8e3c-4c3a8c710a94"
 InfrastructureModels = "2030c09a-7f63-5d83-885d-db604e0e9cc0"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
+ZipFile = "a5390f91-8eb1-5f08-bee0-b1d1ffed6cea"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
 Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+XMLDict = "228000da-037f-5747-90a9-8195ccbf91a5"
 
```

Comment:
can you go through the docs and add documentation for the prvs in the appropriate places (mainly just the places where the components are discussed and the example problem definitions)